### PR TITLE
New goat just dropped, part 2

### PR DESCRIPTION
**Pull request by cuackles**
# Document the changes in your pull request

*new goat, suspicious goat, its like a normal goat but more red and more suspicous, they can also ventcrawl if i didnt fuck up the code
*add said goat to the exotic goat crates
*goat image 
image

# Wiki Documentation

add the sus goat to the goat list in the exotic crate

# Changelog
:cl:  
rscadd: Added new goat and puts it in the exotic goat crate
imageadd: added some icons for the new goat

/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
sus

**Comment by ToasterBiome** [comment]
sus

**Comment by Jarod1200** [comment]
Ah yes  from the popular book by Dr. Sus: One Goat, Two Goat, Red Goat, Dead Goat

**Comment by Hopekz** [comment]
sus

File: yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm [hunk 1]
```diff
@@ -433,3 +433,13 @@
 	icon_dead = "spiffles_dead"
 	faction = list("goat")
 	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/suspicious
+	name = "suspicious goat"
+	desc = "A suspicious looking goat"
```
Comment (Hopekz) [inline yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm]:
```suggestion
	desc = "A suspicious looking goat."
```

**Comment by patpol4** [comment]
good execution

**Review by nesisknesis — APPROVED** [review]

File: yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm [hunk 2]
```diff
@@ -433,3 +433,13 @@
...
 	faction = list("goat")
 	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/suspicious
+	name = "suspicious goat"
+	desc = "A suspicious looking goat."
+	icon = 'yogstation/icons/mob/goats/suspicious_goat.dmi'
+	icon_state = "suspiciusgoat"
+	icon_living = "suspiciusgoat"
+	icon_dead = "suspiciusgoat_dead"
```
Comment (nesisknesis) [inline yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm]: fix spelling in the icon file and here

```suggestion
	icon_state = "suspiciousgoat"
	icon_living = "suspiciousgoat"
	icon_dead = "suspiciousgoat_dead"
```

**Review by nesisknesis — CHANGES_REQUESTED** [review]

**Review by nesisknesis — APPROVED** [review]

**Pull request merged**